teuthology/task/install.py

```diff
@@ -425,6 +429,12 @@ def _update_rpm_package_list_and_install(ctx, remote, rpm, config):
     :param rpm: list of packages names to install
     :param config: the config dict
     """
+    rh_versions=RHVERSION['ceph']
+    if 'rhbuild' in config:
+        if config['rhbuild'] in rh_versions:
```

Comment:
We might want to have an else on this condition and log something if rhbuild was not in rh_versions.